app/components/summary_card/view.rb

```diff
@@ -27,4 +27,8 @@ def rows
   def prevent_action?
     trainee.recommended_for_award? || trainee.awarded? || trainee.withdrawn?
   end
+
+  def row_title(key)
+    key.to_s.downcase.gsub(/ /, "-")
```

Comment:
I think this could possibly be replaced with
```suggestion
    key.parameterize
```

Reply:
Good catch, updated